src/components/Loader/story.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Loader from './Loader';
+import Readme from './README.md';
+
+import { Wrap } from '../../stories/storybook-helpers';
+
+const FlexWrapper = (props) => (
+  <div
+    style={{
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignContent: 'center',
+      alignItems: 'center',
+      marginTop: '100px',
+      marginBottom: '100px',
+    }}
+    {...props}
+  />
+);
+
+storiesOf('Planets/Loader', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add('Overview', () => (
+    <FlexWrapper>
+      <Loader size="md" />
```

Comment:
One more small thing - for the "Overview" stories, we usually instrument all the documented props with "knobs".

Have a look at the Button story for examples if you want:
https://github.com/cision/rover-ui/blob/master/src/components/Button/story.tsx#L42